doc/whats_new/v0.22.rst

```diff
@@ -275,7 +276,18 @@ Changelog
   :pr:`14035` by `Guillaume Lemaitre <glemaitre>`.
 
 :mod:`sklearn.neighbors`
-.............................
+....................
+
+- |Feature| :class:`neighbors.RadiusNeighborsClassifier` now supports
+  predicting probabilities by using predict_proba() and supports more
+  outlier_label options: 'most_frequent', different oulier_labels
+  for multi-outputs.
+  :issue:`9629` by :user:`Wenbo Zhao <webber26232>`.
+
+- |Efficiency| Efficiency improvements for
+  :func:`neighbors.RadiusNeighborsClassifier.prdict` by changing
+  implementation from scipy.stats.mode to numpy.bincount.
```

Comment:
I don't think this is the right description anymore.

Reply:
Isn't it ?

Reply:
I think @jnothman meant something along the lines of,
```rst
- |Efficiency| Efficiency improvements for
  :func:`neighbors.RadiusNeighborsClassifier.predict_proba` by changing
  implementation from scipy.stats.mode to numpy.bincount, and for 
  :func:`neighbors.RadiusNeighborsClassifier.predict` that is now computed as
  an `argmax` of `predict_proba`.
```
?

Reply:
We don't have `neighbors.RadiusNeighborsClassifier.predict_proba` in the current master. It is created in this branch.

How about
`- |Efficiency| Efficiency improvements for
  :func:`neighbors.RadiusNeighborsClassifier.predict` by changing
  implementation from using `scipy.stats.mode` to using `numpy.bincount` in `predict_proba`.`

Reply:
I think implementation in #13783 and #14543 is helpful in `neighbors.KNeighborsClassifier` but not `neighbors.RadiusNeighborsClassifier`, because `neigh_ind` is a 2D array in `neighbors.KNeighborsClassifier` but a nested array (each 1D array has different length) in `neighbors.RadiusNeighborsClassifier`.

Reply:
> We don't have neighbors.RadiusNeighborsClassifier.predict_proba in the current master. It is created in this branch.
> 
> How about
> - |Efficiency| Efficiency improvements for :func:neighbors.RadiusNeighborsClassifier.predictby changing implementation from usingscipy.stats.modeto usingnumpy.bincountinpredict_proba.

Right, of course. But then we can't say that the implementation of predict_proba changed from using `scipy.stats.mode`, as it didn't exist previously. Maybe just saying that "predict is now computed from predict_proba" (and that it's faster) without going in much details.